tests/test_github.py

```diff
@@ -731,6 +767,14 @@ def test_fetch_pulls_from_issues(self):
                                    'X-RateLimit-Remaining': '20',
                                    'X-RateLimit-Reset': '15'
                                })
+        httpretty.register_uri(httpretty.GET,
+                               GITHUB_PULL_REQUEST_1_COMMIT_COMMENTS,
+                               body=pull_request_1_commit_comments,
+                               status=200,
+                               forcing_headers={
+                                   'X-RateLimit-Remaining': '20',
+                                   'X-RateLimit-Reset': '5'
+                               })
```

Comment:
In the test `test_fetch_pulls_from_issues` it seems that no assertion is done over the commit comments data

Reply:
Sure, I'll add it and few more tests!